### DISCUSSION TITLE
Sin permisos para hacer fork

### DISCUSSION THREAD

**Discussion by Ingrzs** [LaunchX-InnovaccionVirtual/CursoIntroPython#255] [Q&A] [answered]
Bueno intento hacer fork al repositorio de curso python y me sale no tienes permiso en esta organización (es decir launch), lo estoy haciendo mal?

**Comment by jstoledano** [comment]
- Puedes descargarlo en zip, que además es el método que se muestra en las capacitación Live. Ya en tu equipo puedes crear un repositorio y controlarlo. Al final es lo mismo.
- Puedes clonarlo, es lo mismo que un fork, pero directamente a tu equipo. En lugar de clonarlo con github (lo que sería un fork) usas git clone.

**Comment by doguedogue** [comment, marked as answer]
No se requiere permiso para el fork. 

Clona y/o descarga el repositorio para trabajar en tu local
